code/modules/client/preference/loadout/loadout_donor.dm

```diff
@@ -1,5 +1,4 @@
 /datum/gear/donor
-	donator_tier = 2
```

Comment:
Um... isn't this going to make all donation items available to donors at a lower level than our patreon says they should be available at?

Reply:
whoops, didn't mean to remove that.